rllib/algorithms/sac/sac_tf_policy.py

```diff
@@ -670,6 +669,65 @@ def setup_early_mixins(
     ActorCriticOptimizerMixin.__init__(policy, config)
 
 
+class ComputeTDErrorMixin:
```

Comment:
Let's put a TODO comment here that this TDErrorMixIn should be unified with the DDPG one, once we sub-class the SAC policy to avoid redundancy here!